docs/user/check-ref/kubernetes_wrapper.rst

```diff
@@ -0,0 +1,243 @@
+        :type **kwargs: dict
+
+        :return: List of services. Typical service has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: endpoints(name=None, **kwargs)
+
+        ::
+
+        Return list of Endpoints.
+
+        :param name: Endpoint name.
+        :type name: str
+
+        :param **kwargs: Endpoint :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of Endpoints. Typical Endpoint has "metadata", and "subsets".
+        :rtype: list
+
+.. py:function:: ingresses(name=None, **kwargs)
+
+        ::
+
+        Return list of Ingresses.
+
+        :param name: Ingress name.
+        :type name: str
+
+        :param **kwargs: Ingress :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of Ingresses. Typical Ingress has "metadata", "spec" and "status".
+        :rtype: list
+
+.. py:function:: statefulsets(name=None, replicas=None, **kwargs)
+
+        ::
+
+        Return list of Statefulsets.
+
+        :param name: Statefulset name.
+        :type name: str
+
+        :param replicas: Statefulset replicas.
+        :type replicas: int
+
+        :param **kwargs: Statefulset :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of Statefulsets. Typical Statefulset has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: replicasets(name=None, replicas=None, **kwargs)
+
+        ::
+
+        Return list of ReplicaSets.
+
+        :param name: ReplicaSet name.
+        :type name: str
+
+        :param replicas: ReplicaSet replicas.
+        :type replicas: int
+
+        :param **kwargs: ReplicaSet :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of ReplicaSets. Typical ReplicaSet has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: deployments(name=None, replicas=None, ready=None, **kwargs)
+
+        ::
+
+        Return list of Deployments.
+
+        :param name: Deployment name.
+        :type name: str
+
+        :param replicas: Deployment replicas.
+        :type replicas: int
+
+        :param ready: Deployment ready status.
+        :type ready: bool
+
+        :param **kwargs: Deployment :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of Deployments. Typical Deployment has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: configmaps(name=None, **kwargs)
+
+        ::
+
+        Return list of ConfigMaps.
+
+        :param name: ConfigMap name.
+        :type name: str
+
+        :param **kwargs: ConfigMap :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of ConfigMaps. Typical ConfigMap has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: persistentvolumeclaims(name=None, phase=None, **kwargs)
+
+        ::
+
+        Return list of PersistentVolumeClaims.
```

Comment:
Add link to https://kubernetes.io/docs/user-guide/persistent-volumes/ (Couldn't find any better link)